### PR TITLE
Don't assume e.Node is nullable

### PR DESCRIPTION
**Pull request by jvitkauskas**
## Proposed changes

-  ~Assume that `e.Node` may be null in first `if` like it's done in the second `if`~
- Don't assume e.Node is nullable

----

:black_nib: I contribute this code under [The Developer Certificate of Origin](../blob/master/contributors.txt).

### CODE REVIEW

**Comment by jvitkauskas** [comment]
Nope, haven't seen any, it's just strange it is assumed to may be null after using as a non-null reference.

File: GitUI/CommandsDialogs/RevisionFileTreeControl.cs [hunk 1]
```diff
@@ -385,7 +385,7 @@ Task ViewGitItemAsync(GitItem gitItem)
 
         private void tvGitTree_BeforeExpand(object sender, TreeViewCancelEventArgs e)
         {
-            if (e.Node.IsExpanded)
+            if (e.Node?.IsExpanded ?? false)
```
Comment (RussKie) [inline GitUI/CommandsDialogs/RevisionFileTreeControl.cs]: Whilst technically `e.Node` can be `null`, I don't believe it is ever `null`.
https://github.com/dotnet/winforms/search?q=TreeViewCancelEventArgs

If anything line:393 should be fixed to remove `?`.

Comment (jvitkauskas) [reply GitUI/CommandsDialogs/RevisionFileTreeControl.cs]: Ok

**Comment by RussKie** [comment]
Please squash and it is good to go

**Comment by jvitkauskas** [comment]
Done

**Comment by jvitkauskas** [comment]
I am already in `contributors.txt`

**Comment by mstv** [comment]
No, I forgot that I had checked out an older branch.

**Pull request merged**